src/main/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -194,6 +209,19 @@ public boolean supportsMetricAccumulationLevel() {
                 return true;
             }
 
+            final String METRICS_EXTENSION = ".insert_size_metrics";
+            final String PDF_EXTENSION = ".insert_size_histogram.pdf";
+
+            final List<String> OUTPUT_EXTENSIONS = CollectionUtil.makeList(
```

Comment:
why `makeList` instead of `Arrays.asList`?
```suggestion
            final List<String> OUTPUT_EXTENSIONS = Arrays.asList(
```